### PR TITLE
Add basic doc files

### PR DESCRIPTION
**Pull request by BottleRocket-Colin**
Added license and readme files for Open Source project documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#LaunchPad Quick Start for Android
+
+The LaunchPad project is intended to be an aid for helping you bring your Android projects to market quickly without the need to reinvent the wheel.  It is open source and free to use.
+
+## LaunchPad Compose
+This library provides a collection of Compose widgets and utility functions with the goal of making coding in Compose that much easier.
+
+## Contributing
```
Comment (jaredstockton) [inline README.md:8]: @BottleRocket-Colin Let's add a setup or usage section that basically shows how to add the library to the appropriate build gradle files. Feel free to modify the starter markdown I have below

## Setup
1. You'll need to add a dependency on [jitpack](https://jitpack.io/) to your project build.gradle.kts file. Make sure it is below primary maven repositories such as `google()` or `mavenCentral()` as it should have lower precedence in Gradle's dependency resolution process:
```kotlin
allprojects {
    repositories {
       ... // likely already have google() and mavenCentral() above
        maven(url = "https://jitpack.io")
    }
}
```

3. You'll need to add the following line to your module's build.gradle.kts:
```kotlin
dependencies {
    ...
    implementation("com.github.BottleRocketStudios:Android-LaunchPad-Compose:0.2.0")
}
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This project is open source and we welcome updates from all devs.  Look to [`CONTRIBUTING.md`](./CONTRIBUTING.md) for information on how to help out.
+
+## What's in this library
+
+### Convenience Functions
+
+* Check out ThemeExtensions.kt for a collection of easy to use functions for making your Compose Themes easier to read.
```
Comment (jaredstockton) [inline README.md]: @BottleRocket-Colin nit: Consider either:

1. Linking to the Github file link - [ThemeExtensions.kt](https://github.com/BottleRocketStudios/Android-LaunchPad-Compose/blob/main/launchpad-compose/src/main/java/com/bottlerocketstudios/launchpad/compose/ThemeExtensions.kt) OR
2. Use the inline code style around the filename like `ThemeExtensions.kt`

**Review by jaredstockton — CHANGES_REQUESTED** [review]
@BottleRocket-Colin Looking good! A few small comments to address prior to approval :)

File: README.md (hunk 1 above)
Comment (BottleRocket-Colin) [reply README.md:8]: Added a "Usage" section at bottom

**Review by jaredstockton — APPROVED** [review]
Appreciate the work here!

**Pull request merged**